java/src/main/java/io/opentracing/contrib/TracerLoader.java

```diff
@@ -0,0 +1,38 @@
+package io.opentracing.contrib;
+
+import io.opentracing.NoopTracerFactory;
+import io.opentracing.Tracer;
+
+import java.util.Iterator;
+import java.util.ServiceLoader;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by wusheng on 2016/12/21.
```

Comment:
Let's make a good comment here showing how this fits in, and let's also add something to the README.